src/Console/Command/ProjectionistCommand.php

```diff
@@ -16,8 +25,50 @@ public function __construct(
         parent::__construct();
     }
 
-    protected function projectorCriteria(): ProjectorCriteria
+    protected function configure(): void
+    {
+        $this
+            ->addOption(
+                'id',
+                null,
+                InputOption::VALUE_IS_ARRAY,
+                'The maximum number of runs this command should execute'
+            );
+    }
+
+    protected function projectorCriteria(InputInterface $input): ProjectorCriteria
```

Comment:
I'm not sure about passing here the whole `InputInterface` into it. We gain alot of power with this and possibilities but could it also backfire? WDYT?